### ISSUE TITLE
CreationTimestamp is garbled

### ISSUE THREAD

**Issue by tamalsaha** [appscode/json2yaml#1]
```
// to be run using nodejs
var json2yaml = require('../src/json2yaml').json2yaml;
function test(obj) {
  console.log('TESTING:');
  console.log(obj);
  console.log('OUTPUT:');
  console.log(json2yaml(obj));
  console.log('------------------------');
}

test({
  "status": "True",
  "lastUpdateTime": "2017-03-02T05:41:47Z",
  "schedule": "every @6m"
});

```

**Comment by mirshahriar** [comment]
cronExpression: "@every\x202m"

**Comment by tamalsaha** [comment]
Some background: http://symfony.com/doc/current/components/yaml/yaml_format.html#strings

**Comment by tamalsaha** [comment]
- https://github.com/jeremyfa/yaml.js/blob/2a19bf438d21d2adab256b6397a1f773d5e70ace/dist/yaml.js#L320

 - https://github.com/jeremyfa/yaml.js/blob/develop/lib/Escaper.js
 - https://github.com/jeremyfa/yaml.js/blob/develop/lib/Pattern.js

**Comment by tamalsaha** [comment]
Fixed #3
